streams/src/test/java/org/apache/kafka/streams/integration/InternalTopicIntegrationTest.java

```diff
@@ -69,42 +70,57 @@ public static void startKafkaCluster() throws Exception {
     }
 
     /**
-     * Validates that any state changelog topics are compacted
-     * @return true if topics have a valid config, false otherwise
+     * Check if the given topic exists.
+     *
+     * @param topic
+     * @param config
+     * @return
      */
-    private boolean isUsingCompactionForStateChangelogTopics() {
```

Comment:
I think we should, and with #1792 maybe this test needs to be updated a bit again, but still by default (user does not overwrite anything) the changelog should be compacted.